stdlib/unittest/runner.pyi

```diff
@@ -2,13 +2,21 @@ import sys
 import unittest.case
 import unittest.result
 import unittest.suite
-from _typeshed import Incomplete
 from collections.abc import Callable, Iterable
-from typing import TextIO
-from typing_extensions import TypeAlias
+from typing import Any, TextIO
+from typing_extensions import Never, TypeAlias
 
 _ResultClassType: TypeAlias = Callable[[TextIO, bool, int], unittest.result.TestResult]
 
+# Note: doesn't actually inherit TextIO, but re-exposes all methods of the stream passed to __init__
```

Comment:
I'm not sure we should even use `TextIO` as annotation for `stream`. This annotation most likely predates the introduction of protocols (I didn't check) and we should use a tight protocol instead. `_WritelnDecorator` could then either just add the methods of that protocol or we could make `TextTestRunner` generic over the stream, although I'm not sure that that's worth it.

Reply:
> we could make `TextTestRunner` generic over the stream, although I'm not sure that that's worth it.

I don't think there's any way to fill a class attributes from generics right?
``py
_T= TypeVar("_T", bound=type)
class _WritelnDecorator(_T): ...
```

The closest I can do is copy all the methods from TextIO